tiledb/sm/subarray/range_subset.cc

```diff
@@ -131,8 +131,18 @@ RangeSetAndSuperset::RangeSetAndSuperset(
     ranges_.emplace_back(superset);
 }
 
-Status RangeSetAndSuperset::sort_ranges(ThreadPool* const compute_tp) {
-  return impl_->sort_ranges(compute_tp, ranges_);
+void RangeSetAndSuperset::sort_and_merge_ranges(
+    ThreadPool* const compute_tp, bool merge) {
+  // Return if there's not at least 2 ranges
+  if (ranges_.size() < 2) {
```

Comment:
Is this necessary? If I call sort_ranges or merge_ranges with one range, do they crash or take a long time?

Reply:
if `ranges.empty()`, the code will segfault. It just seemed _unnecessary_ to call `sort_ranges` or `merge_ranges` on only a single range but it doesn't cause a problem. I changed this to only return `if (ranges.empty())`